src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeRates.tsx

```diff
@@ -209,7 +209,7 @@ export const useGetTradeRates = () => {
       // Clear the slice before asynchronously generating the input and running the request.
       // This is to ensure the initial state change is done synchronously to prevent race conditions
       // and losing sync on loading state etc.
-      dispatch(tradeQuoteSlice.actions.clear())
+      dispatch(tradeQuoteSlice.actions.clearTradeQuotes())
```

Comment:
Don't clear the whole slice - `clearTradeQuotes` keeps `activeQuoteMeta` i.e the selected quote. This ensures that when re-mounted, `<TradeInput />` doesn't lose track of the user-selected quote if any